Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php

```diff
@@ -40,8 +40,8 @@ public function fix(\SplFileInfo $file, $content)
 
             // ignore inline docblocks
             if (
+                $prevToken->equalsAny(array(';', '{')) ||
                 ($prevToken->isWhitespace(array('whitespaces' => " \t")) && !$tokens[$index - 2]->isGivenKind(T_OPEN_TAG))
-                || $prevToken->equalsAny(array(';', '{'))
```

Comment:
It was proposal from analyser (costs estimation), as I understood semantics remains unchanged.
I'll revert if there are any doubts.